### PR TITLE
update package naming advice

### LINKED ISSUE #697: {available} package or pak
https://pak.r-lib.org/reference/pkg_name_check.html

### PR DESCRIPTION
**Pull request by maelle**
Fix #697

### CODE REVIEW

File: pkg_building.es.Rmd [hunk 1]
```diff
@@ -15,7 +15,7 @@ Para leer por qué vale la pena enviar un paquete a rOpenSci siguiendo las recom
 
 - Recomendamos fuertemente elegir nombres cortos y descriptivos en minúsculas.
   Si tu paquete se conecta con uno o más servicios comerciales, asegúrate de que el nombre no infringe las recomendaciones de la marca.
-  Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando el [paquete `available`](https://github.com/ropenscilabs/available).
+  Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando la función [`pak::pkg_name_check()`](https://pak.r-lib.org/reference/pkg_name_check.html); también utiliza un motor de búsqueda porque asi verías si es ofensivo en otras idiomas que inglés.
```
Comment (yabellini) [inline pkg_building.es.Rmd:18]:
```suggestion
  Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando la función [`pak::pkg_name_check()`](https://pak.r-lib.org/reference/pkg_name_check.html); también utiliza un motor de búsqueda para comprobar si el nombre es ofensivo en otros idiomas distinto al idioma original.
```

Comment (yabellini) [reply pkg_building.es.Rmd:18]: I changed the text thinking the function pck_name_check looks for names in English.  People can use other languages to name their packages.  Let me know if this is not the case, and I will change the text again.

**Review by yabellini — COMMENTED** [review]
I left some comments.

**Review by mpadge — APPROVED** [review]

File: pkg_building.es.Rmd (hunk 1 above)
Comment (maelle) [reply pkg_building.es.Rmd:18]: So, they should still use the function for checking the name is available. I edited your suggestion, does that make sense?

Comment (yabellini) [reply pkg_building.es.Rmd:18]: Yes!

**Pull request merged**